hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsCriteria.java

```diff
@@ -248,6 +250,28 @@ public void addTag(String name, String value) {
         tags.put(name, value);
     }
 
+    public String getTagQuery() {
+        return tagQuery;
+    }
+
+    /**
+     * @param tagQuery return alerts with *any* of the tags expecified by the tag expression language:
+     *
+     *        <tag_query> ::= ( <expression> | "(" <object> ")" | <object> <logical_operator> <object> )
+     *        <expression> ::= ( <tag_key> | <not> <tag_key> | <tag_key> <boolean_operator> <tag_value> |
+     *               <tag_key> <array_operator> <array> )
+     *        <not> ::= [ "NOT" | "not" ]
+     *        <logical_operator> ::= [ "AND" | "OR" | "and" | "or" ]
+     *        <boolean_operator> ::= [ "==" | "!=" ]
+     *        <array_operator> ::= [ "IN" | "NOT IN" | "in" | "not in" ]
+     *        <array> ::= ( "[" "]" | "[" ( "," <tag_value> )* )
+     *        <tag_key> ::= <identifier>                                // Tag identifier
+     *        <tag_value> ::= ( "'" <regexp> "'" | <simple_value> )     // Regular expression used with quotes
+     */
```

Comment:
+1
I don't have a preference, either. I introduced "key" nomenclature when I imported the metrics g4.
"name" is fine I will sync with that.